src/meta/src/hummock/compaction/level_selector.rs

```diff
@@ -69,10 +67,7 @@ pub struct DynamicLevelSelector {
 impl Default for DynamicLevelSelector {
     fn default() -> Self {
         let config = Arc::new(CompactionConfig::default());
```

Comment:
Why not store `ProstCompactionConfig` directly

Reply:
It could be.

Reply:
It's not convenient to add new methods for ProstCompactionConfig, so I'd like to keep CompactionConfig.

Reply:
No.... Config is a lightweight structure， we do no need add too many methods for it

Reply:
Fixed.